src/resolvers/OrganizationResolver.ts

```diff
@@ -43,6 +43,14 @@ class OrganizationResolver {
     return OrganizationRepo.getOrganizationBySlug(slug);
   }
 
+  @Query((_return) => Organization, {
+    nullable: true,
+    description: `Returns the <Organization> given a <slug> and <accessCode>.`,
+  })
+  async checkAccessCode(@Arg('accessCode') accessCode: string, @Arg('slug') slug: string) {
+    return OrganizationRepo.checkAccessCode(accessCode, slug);
+  }
```

Comment:
Maybe we can set up some sort of session token auth here as well -- we don't necessarily need to use the graphQL directly to do auth, we can also just create an express endpoint for this that sends over a session token and then create a middleware for the protected queries and mutations

Reply:
We could try something like this, but our concern with using session tokens was if someone logs in and then 1 month later their session expires and they forget their password, we want to avoid a situation when an org forgets their password since we don't have any email recovery system set up.

Reply:
If that's your goal you can just ensure the decoded token is valid and then if the date has expired issue a new refreshed token